PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -26,7 +26,7 @@
         android:id="@+id/mainListView"
         android:layout_width="@dimen/main_list_view_width"
         android:layout_height="@dimen/main_list_view_height"
-        android:layout_alignParentRight="true"
+        android:layout_toRightOf="@+id/avatarLayout"
```

Comment:
You should only use `@+id` when defining a new id. Since here you are referring to one, you should use `@id` instead